### PR TITLE
Removed the Discussion answered Highlight Badge

### PR DESCRIPTION
**Pull request by MitkoVtori**
Removed the Discussion answered Highlight Badge since it doesn't exist anymore.

### CODE REVIEW

**Comment by NathanBollard** [comment]
This pull request should be merged.

**Review by HackerstoyanovBG — APPROVED** [review]
That's clean. GitHub actually removed the  badge. That should be merged.

File: README.md [hunk 1]
```diff
@@ -65,7 +65,6 @@
 | Badge | Name | How to get |
 | :-: | :-: | :-: |
 | ![pro-black](https://user-images.githubusercontent.com/65187002/173065669-d1fdb5a7-8895-43cc-8dea-72a511a37e86.svg#gh-light-mode-only) ![pro-white](https://user-images.githubusercontent.com/65187002/173065531-57dbf8b1-7eb7-4d46-81bf-f2d18c7c9112.svg#gh-dark-mode-only) | Pro | Use [GitHub Pro](https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro) |
-| ![discussion-answered-dark](https://user-images.githubusercontent.com/65187002/173078083-15a75f15-b040-4a92-8d70-561a206d9fd9.svg#gh-dark-mode-only)![discussion-answered-light](https://user-images.githubusercontent.com/65187002/173078106-28bea542-4620-46ee-837d-defda3e44ca6.svg#gh-light-mode-only) | Discussion answered | Have  your reply to a discussion marked as the answer |
```
Comment (MitkoVtori) [inline README.md:68]: If you want, I can instead of removing it, to add one "x" as no longer earnable.

**Review by MitkoVtori — COMMENTED** [review]
Hope you answer this pull request soon!

File: README.md (hunk 1 above)
Comment (CondratKenasil) [reply README.md:68]: That also would be great

**Review by CondratKenasil — APPROVED** [review]
This should be merged

**Review by drknzz — APPROVED** [review]

**Review by NathanBollard — COMMENTED** [review]
ayo

**Pull request merged**